source/sessions/snapshot-sessions.md

```diff
@@ -110,11 +116,41 @@ Snapshot reads and causal consistency are mutually exclusive. Therefore if `snap
 `causalConsistency` must be false. Client MUST throw an error if both `snapshot` and `causalConsistency` are set to
 true. Snapshot reads are supported on both primaries and secondaries.
 
+### snapshotTime
+
+Applications set `snapshotTime` when starting a snapshot session to specify the desired snapshot time.
+
+Note that the `snapshotTime` property is optional. The default value of this property is null.
+
+Client MUST throw an error if `snapshotTime` is set and `snapshot` is not set to true.
+
+Note that when parsing `snapshotTime` from `sessionOptions` for
```

Comment:
Is this a good place to describe this behaviour?

Reply:
I would say we have to mention this in `tests/README.md` file.

Reply:
Makes sense